Fixed #22774 Can't update totals for credit note

gaurav-473 opened the pull request:
Fixed #22774 Can't update totals for credit note
### Preconditions (*)
1. Magento Open Source 2.3.1

### Steps to reproduce (*)
1. Select an invoice
2. Click "Credit Memo"
3. Add value to `Refund Shipping `, `Adjustment Refund` or `Adjustment Fee` field

### Expected result
1. Should be able to update totals using the "Update Totals" button

![M2 2 8](https://user-images.githubusercontent.com/2035088/57310996-ba2de300-70e2-11e9-938f-8dcf8c2174e9.png)


### Actual result
1. There is no way to update the total (without changing the quantity and then changing it back again)

![M2 3 1](https://user-images.githubusercontent.com/2035088/57311003-bef29700-70e2-11e9-8848-5ed613305810.png)

gaurav-473 committed:
Fixed #22774

gaurav-473 committed:
Fixed #22774

gaurav-473 committed:
Fixed #22774

gaurav-473 committed:
Merge branch '2.3-develop' into 2.3-developPr22774

swnsma commented:
Hi @guarinogabriel,
Thank you for your contribution!

As I can see, you are trying to add changes which had been done in 2.2-develop with commit 451f7b9f317db5a27d3d9c541cc3c1678cc169f5 in scope of issue #11798 and these changes were not Forwardported to 2.3-develop.

May I ask you to create PR with all changes from the 451f7b9f317db5a27d3d9c541cc3c1678cc169f5 ?

swnsma reviewed (changes requested):
Hi @guarinogabriel,
Thank you for your contribution!

As I can see, you are trying to add changes which had been done in 2.2-develop with commit 451f7b9 in scope of issue #11798 and these changes were not Forwardported to 2.3-develop.

May I ask you to create PR with all changes from the 451f7b9 ?

gaurav-473 committed:
Fixed #22774

gaurav-473 committed:
Fixed #22774

gaurav-473 committed:
Fixed #22774

gaurav-473 committed:
Fixed #22774

gaurav-473 committed:
Fixed #22774

gaurav-473 committed:
Merge branch '2.3-develop' into 2.3-developPr22774

gauravagarwal1001 commented:
Hi @swnsma,
I updated all changes of commit 451f7b9.

swnsma commented on app/code/Magento/Sales/Block/Adminhtml/Order/Creditmemo/Create/Adjustments.php:
```diff
@@ -4,9 +4,7 @@
  * See COPYING.txt for license details.
  */
 namespace Magento\Sales\Block\Adminhtml\Order\Creditmemo\Create;
-
```
Can we avoid such changes?

gauravagarwal1001 replied:
Sure I removed empty lines.

gaurav-473 replied:
There must be one blank line after the namespace declaration.

swnsma commented on app/code/Magento/Sales/view/adminhtml/templates/order/creditmemo/create/totals/adjustments.phtml:
```diff
@@ -39,7 +40,6 @@
                    id="adjustment_negative"/>
             <script>
                 require(['prototype'], function(){
-
```
Can we avoid such changes?

gauravagarwal1001 replied:
Sure I removed empty lines

swnsma reviewed (changes requested):
Hi @gauravagarwal1001 
Thank you for your changes.

Could you please:
1) Dismiss remove of such amount empty lines
2) Check and adjust changes according to static tests build fail.
3) Check and adjust failed functional tests.

gaurav-473 committed:
Add blank lines according to PHP Code Sniffer

There must be one blank line after the namespace declaration
There must be one blank line after the last USE statement

swnsma commented:
@gauravagarwal1001,  I am closing this PR now due to inactivity.
Please reopen and update if you wish to continue.
Thank you for the collaboration!
